requirements.txt

```diff
@@ -1,8 +1,13 @@
-Flask-Cache
-Flask-WTF==0.11
+python==2.7.12
+python-dev==2.7
```

Comment:
As there was no version number prior to this commit I just used the installation default ones and checked that it worked. If you have any version number already supported I'd be glad to have them :)